### DISCUSSION TITLE
Convert sado.space website to Valkyr ?

### DISCUSSION THREAD

**Discussion by msmalley** [sadoprotocol/sado.space#1] [Ideas]
This would provide an isolated environment for me to review Valkyr and for you to test the SADO API ,,,

It would also enforce you to create proper documentation for Valkyr as each relevant module was used on sado.space ...

**Comment by kodemon** [comment]
I think there is definitely potential for us to employ at least a decent subset of Valkyr for sado.space

 - [https://www.solidjs.com/](SolidJS)
   - [@valkyr/db](https://github.com/cmdo/valkyr/tree/main/packages/db) would give us indexedb wrapper mongodb like client side data store.
   - [@valkyr/route](https://github.com/cmdo/valkyr/tree/main/packages/router) for routing.
   - [@valkyr/solid](https://github.com/cmdo/valkyr/tree/main/packages/solid) provides a controller layer to separate data logic and component code. It also comes with plugins for nested routing and is extensible. Also comes with a base solution for Forms.
   
I can set up a baseline sample if you want to have a look at how it would look for sado.space.

---

I am not sure about EventSourcing aspect yet if its something we may want to entertain. But I think for velocity we may want to stick with a more minimal solution first 🤔

**Reply by msmalley** [reply]
My primary objective is to get the (sado.space) project under your control as quickly as possible.
My only requirement is that things (or things being used) are properly documented.

Beyond that, I am open (on this project) to explore ANY recommendations or preferences you have ...

**Reply by msmalley** [reply]
Let me quickly update the sado.space REPO first - do you have access to that already?

**Reply by kodemon** [reply]
I have access 👍🏻

**Reply by kodemon** [reply]
> My only requirement is that things (or things being used) are properly documented.

Yup, @valkyr documentation is basically the current priority for valkyr as well so it will be actively worked on once my focus shifts back into client side space.

**Comment by kodemon** [comment]
Conversion has been made and development is proceeding at this point.
